update_freq: 30.0
initial_mode: 7 #0
joy_override_buttion: 7
joy_override_timeout: 15.0
joy_reset_buttion: 6
joy_setyaw_button: 4
idle_timeout: 10
servo_desired_depth: 2.5
#target_height: 0.5
#target_dist_ground: 0.75
target_height: 1.5
target_dist_ground: 1.0
stale_video_timeout: 10.0
enable_camera_control: true
#max_camera_tilt_deg: 0.0
max_camera_tilt_deg: 45
#search_alt: 9.0
search_alt: 12.0
perform_search_action: true
#perform_search_action: false
flow_queue_size: 15
flow_threshold: 5.5
flow_min_votes: 10
